Timelines/Timeline_onScroll_Animation_1/app.js

```diff
@@ -0,0 +1,19 @@
+window.addEventListener('scroll', () => {
+  let contents = document.querySelectorAll('.child-container');
+  let logoImages = document.querySelectorAll('.logoImage');
+  contents.forEach((content) => {
+    let contentPosition = content.getBoundingClientRect().top;
+    let screenPosition = window.innerHeight;
+    contentPosition < screenPosition
+      ? content.classList.add('active')
```

Comment:
Avoid using ternary operator if possible since it makes the code difficult to read and understand

Reply:
So should i change it to normal if/else syntax or should add a comment there ?
@Ovenoboyo

Reply:
Preferably normal if-else